.pre-commit-config.yaml

```diff
@@ -1,11 +1,28 @@
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
+    rev: v4.3.0
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
     -   id: check-yaml
 -   repo: https://github.com/psf/black
-    rev: 19.3b0
+    rev: 22.6.0
     hooks:
     -   id: black
+-   repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+    -   id: isort
+        args: ["--profile", "black", "--filter-files"]
+        name: isort (python)
+# -   repo: https://github.com/doublify/pre-commit-rust
+#     rev: v1.0
+#     hooks:
+#     -   id: fmt
+#     -   id: cargo-check
+#     -   id: clippy
+- repo: https://github.com/python-poetry/poetry
+  rev: '1.2.0rc1'
+  hooks:
+    - id: poetry-check
+    # - id: poetry-lock
```

Comment:
Do we need an issue to track getting `poetry-lock` working properly, or is it addressed in the other pull request?

Reply:
Let's open an issue for that

Reply:
Issue #156 is for tracking this